sample.env

```diff
@@ -1,305 +1,421 @@
+SMTP_USERNAME = ''
+SMTP_PASSWORD = ''
+SMTP_EMAIL = ''
+
+GCP_PROJECT_ID=
+
+# If ENABLE_EMAILER is 'False', set the following environment variables
+COUNT_MESSAGE_TYPES = 'bsm'
+ODE_KAFKA_BROKERS = ${DOCKER_HOST_IP}:9092
+
+# EITHER "MONGODB" or "BIGQUERY"
+COUNT_DESTINATION_DB = 'MONGODB'
+
+# MONGODB REQUIRED VARIABLES
+INPUT_COUNTS_MONGO_COLLECTION_NAME = ''
+OUTPUT_COUNTS_MONGO_COLLECTION_NAME = ''
+
+KAFKA_BIGQUERY_TABLENAME=
+
+#### ---- rsu_status_check
+
+# Services that can be toggled on or off
+# 'True' or 'False' are the only legal values
+
+# Toggles monitoring of RSU online status
+RSU_PING=True
+
+# Fetches ping data from Zabbix - alternatively the service will ping the RSUs on its own
+# Only used when RSU_PING is 'True'
+ZABBIX=False
+
+# Fetches SNMP configuration data for all RSUs
+RSU_SNMP_FETCH=True
+
+# Zabbix endpoint and API authentication
+# Only used when ZABBIX is 'True'
+ZABBIX_ENDPOINT=
+ZABBIX_USER=
+ZABBIX_PASSWORD=
+
+# Customize the period at which the purger will determine a ping log is too old and will be deleted
+# Number of hours
+STALE_PERIOD=24
+
+#### ---- jpo_iss_health_check
+
+# Key Storage
+## Type of key storage, options: gcp, postgres
+STORAGE_TYPE=postgres
+
+# ISS Account Authentication
+ISS_API_KEY=
+ISS_API_KEY_NAME=
+ISS_PROJECT_ID=
+ISS_SCMS_TOKEN_REST_ENDPOINT=
+ISS_SCMS_VEHICLE_REST_ENDPOINT=
+
+## Postgres Storage (Required if STORAGE_TYPE=postgres)
+### Table name to store keys
+ISS_KEY_TABLE_NAME=
+
+#### ---- firmware_manager_upgrade_runner
+
+BLOB_STORAGE_PROVIDER=DOCKER
+BLOB_STORAGE_BUCKET=
+
+# If "BIGQUERY", set the location of the GCP service account key attached as a volume
+GOOGLE_APPLICATION_CREDENTIALS='./resources/google/sample_gcp_service_account.json'
+
+## Docker volume mount point for BLOB storage (if using Docker)
+HOST_BLOB_STORAGE_DIRECTORY=./local_blob_storage
+
+## Maximum retry limit for performing firmware upgrades
+FW_UPGRADE_MAX_RETRY_LIMIT=3
+
+#### ---- firmware_manager_upgrade_scheduler
+
+FIRMWARE_MANAGER_UPGRADE_RUNNER_ENDPOINT=http://${DOCKER_HOST_IP}:8090
+
+######## -------- "cvmgr_mongo" Docker Profile Services
+# Run MongoDB database for CV message counts, used by the jpo_count_metric addon
+# Requires: None
+# Services: cvmanager_mongo, cvmanager_mongo_setup
+# - cvmanager_mongo
+#   - MongoDB instance to store jpo_count_metric counts
+# - cvmanager_mongo_setup
+#   - Configure and initialize MongoDB instance
+
+#### ---- cvmanager_mongo
+
+MONGO_ADMIN_DB_USER=root
+MONGO_ADMIN_DB_PASS=root
+
+#### ---- cvmanager_mongo
+
+MONGO_DB_NAME="ode"
+MONGO_CV_MANAGER_DB_USER=api
+MONGO_CV_MANAGER_DB_PASS=api
+
+# If running MongoDB locally with docker-compose-mongo.yml uncomment the following line
+MONGO_COLLECTION_TTL=7 # days
+
+# If true sample data will be inserted into the MongoDB CVCounts, V2XGeoJson, and OdeSsmJson collections.
+INSERT_SAMPLE_DATA=true
+
+
+######## -------- "obu_ota" Docker Profile Services
+# Run OBU over the air update microservices
```

Comment:
Lets make this "over-the-air" since that is a standard way to write this phrase.